### PR TITLE
added relationships to autofocus

### PR DESCRIPTION
**Pull request by orhovy**
Added relationships to AutoFocus enrichments commands

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35561

## Description
Added relationships to AutoFocus enrichments commands

## Screenshots

https://user-images.githubusercontent.com/58938354/115993944-aafd1400-a5dd-11eb-8497-f10b8bd59809.mov

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by orhovy** [comment]
`Campaign` indicator type still does not exist,  Maybe should wait until the indicator type will be created.

**Comment by orhovy** [comment]
`Campaign` indicator type still does not exist,  Maybe should wait until the indicator type will be created.

**Review by bakatzir — COMMENTED** [review]
1. Let's indeed wait for https://github.com/demisto/etc/issues/338062 - @gal-berger do you have an ETA? we need the `campaign` indicator.
2. please add the outputs to the readme
3. please add UT to cover the different relationships type creation that are available in `RELATIONSHIP_TYPE_BY_TAG_CLASS_ID`

**Review by bakatzir — APPROVED** [review]
nice work. let's wait for the decision on the `Campaign` IOC prior to merging.

**Comment by yuvalbenshalom** [comment]
Campaign indicators type is merged.
this is good to go when autofocus tests will be back to normal.

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py
Comment (yuvalbenshalom) [inline Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:35]:
```suggestion
    2: {'entity_b_type': 'Campaign',
```

**Review by yuvalbenshalom — APPROVED** [review]
just remove the comment on Campaign

**Pull request merged**
